.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: rust
 rust:
-    - 1.35.0
+  - stable
+  - beta
+  - nightly
```

Comment:
We deliberately picked a fixed version, so CI doesn't randomly break. Instead, we'd regularly update it manually, and in the same PR also add any Clippy or Rustfmt changes.
Feel free to update it to the current 1.38.0.

Reply:
I suggest keep stable, beta and nightly alongwith 1.38.0 to make sure we are good with most people's environments

Reply:
I suggest keeping stable, beta and nightly alongwith 1.38.0 to make sure we are good with most environments and can discover any problem as early as possible

Reply:
For now `allow_failures` is set for nightly, stable and beta looks quite stable to me though

If you'd like we can be more merciful